### PR TITLE
Add GitHub actions

### PR DESCRIPTION
**Pull request by zsluedem**

### CODE REVIEW

File: bin/bundler.rs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 use aa_bundler::{
-    bundler::bundler::Bundler,
+    bundler::Bundler,
```
Comment (zsluedem) [inline bin/bundler.rs:2]: fix  https://rust-lang.github.io/rust-clippy/master/index.html#module_inception

File: bin/bundler-rpc.rs [hunk 2]
```diff
@@ -2,7 +2,6 @@ use anyhow::Result;
 use clap::Parser;
 use jsonrpsee::{core::server::rpc_module::Methods, server::ServerBuilder, tracing::info};
 use std::future::pending;
-use tracing_subscriber;
```
Comment (zsluedem) [inline bin/bundler-rpc.rs:5]: unused.

File: src/uopool/server.rs [hunk 3]
```diff
@@ -2,6 +2,6 @@ mod types {
     tonic::include_proto!("types");
 }
 
-pub mod server {
+pub mod uopool_server {
```
Comment (zsluedem) [inline src/uopool/server.rs:5]: fix https://rust-lang.github.io/rust-clippy/master/index.html#module_inception

File: src/uopool/mod.rs [hunk 4]
```diff
@@ -26,6 +26,12 @@ impl UserOperationPool {
     }
 }
 
+impl Default for UserOperationPool {
+    fn default() -> Self {
+        Self::new()
+    }
+}
```
Comment (zsluedem) [inline src/uopool/mod.rs:33]: fix https://rust-lang.github.io/rust-clippy/master/index.html#new_without_default

**Review by Vid201 — APPROVED** [review]

**Comment by Vid201** [comment]
lgtm :rocket:

**Pull request merged**